src/theory/theory_engine.cpp

```diff
@@ -396,6 +404,14 @@ void TheoryEngine::check(Theory::Effort effort) {
       d_tc->resetRound();
     }
 
+    if (d_partitionGen != nullptr)
+    {
+      TrustNode tl = d_partitionGen->check(effort);
+      if (!tl.isNull()){
```

Comment:
Format.

Reply:
I'm really confused because I am using clang format every time. 
git add <whatever>
git-clang-format
git add <formatted stuff>
commit
push

Is this wrong?

Reply:
I think everyone has a different configuration of how they clang format, I use `git clang-format master` where `master` is the main branch of my fork.